package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
Hmm... And you're sure it wasn't working before? Could you try removing the `--coverage` from this line and showing me the output for that? The reason I'm so skeptical is because that's not a documented flag for `ava` and honestly doesn't really make sense to me.